src/cls/rgw/cls_rgw_ops.h

```diff
@@ -385,21 +385,23 @@ struct rgw_cls_list_op
   uint32_t num_entries;
   std::string filter_prefix;
   bool list_versions;
+  bool want_header;
```

Comment:
Not sure whether this is absolutely necessary; perhaps it could be deduced from zeroed `num_entires`.